src/main/frontend/components/query_table.cljs

```diff
@@ -63,12 +63,14 @@
   (let [p-desc? (get-in current-block [:block/properties :query-sort-desc])
         desc? (if (some? p-desc?) p-desc? true)
         p-sort-by (keyword (get-in current-block [:block/properties :query-sort-by]))
+        nlp-date? (get-in current-block [:block/properties :query-nlp-date])
```

Comment:
Starting with https://github.com/logseq/logseq/pull/6105, we started putting properties under namespaces. Could we do that here e.g. `:logseq.query/nlp-date`? By namespacing we avoid accidental clashes with user properties